### PR TITLE
ACP2E-2239: Documentation improvement

### PR DESCRIPTION
**Pull request by thiaramus**
## Purpose of this pull request

This pull request (PR) improves the integration test documentation based on the feedback from developers.

## Affected pages

- https://developer.adobe.com/commerce/testing/guide/integration/attributes/data-fixture

## Links to Magento Open Source code

- ...

### CODE REVIEW

**Comment by thiaramus** [comment]
Keeping this PR as a Draft while we're working on an internal review.

File: src/pages/guide/integration/attributes/data-fixture.md [hunk 1]
```diff
@@ -186,6 +180,7 @@ Data Fixture is a PHP class that implements `Magento\TestFramework\Fixture\DataF
 1. Data Fixture MUST NOT depend on another fixture.
 1. Data Fixture SHOULD be implemented using service APIs.
 1. Data Fixture SHOULD have dynamic default data to allow generating unique fixtures.
+1. Validation should be handled by the service class and not in the data fixture class.
```
Comment (bubasuma) [inline src/pages/guide/integration/attributes/data-fixture.md:183]: I think we should rephrase this to : Data Fixture MUST NOT handle input validation. Such validation should be handled by the service class.

**Review by bubasuma — CHANGES_REQUESTED** [review]

File: src/pages/guide/integration/attributes/data-fixture.md (hunk 1 above)
Comment (thiaramus) [reply src/pages/guide/integration/attributes/data-fixture.md]: Corrected. I also added the scope edits.

**Review by bubasuma — APPROVED** [review]

**Review by jeff-matthews — APPROVED** [review]

**Comment by jeff-matthews** [comment]
Thanks for keeping the testing docs fresh @thiaramus & @bubasuma!
